platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/MerkleDbDataSource.java

```diff
@@ -621,6 +599,8 @@ public VirtualLeafRecord<K, V> loadLeafRecord(final K key) throws IOException {
             return null;
         }
 
+        //        assert validLeafPathRange.withinRange(path);
```

Comment:
Please remove it if it's no longer relevant

Reply:
Removed.